Source/SteveSloth/Private/MyEnemy.cpp

```diff
@@ -0,0 +1,39 @@
+/****************************************************************************************
+
+Copyright: SteveSloth
+
+Name: Aidan
+
+Script: MyEnemy.cpp
+
+Date: April 23. 2024
+
+Description: This is the base class for all Enemies.
+
+TODO:
+
+Known Bugs:
+
+TODO:
+
+Known Bugs:
+****************************************************************************************/
+
+
+#include "MyEnemy.h"
+
+MyEnemy::MyEnemy()
+{
+	MaxHealth = 0;
+	CurrentHealth = 0;
+	Damage = 0;
+	MovementState = Idle;
+	AttackType = Melee;
```

Comment:
AttackType is an Enum not a variable. You cannot set AttackType to = Melee
you need to create a variable of type AttackType